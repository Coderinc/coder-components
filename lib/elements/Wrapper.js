import styled from 'styled-components';

export default styled.div`
  box-sizing: border-box;
  width: 100%;
  height: 100%;
  background-color: ${props => props.theme.primaryBackground};
  color: ${props => props.theme.primaryFont};
`;
