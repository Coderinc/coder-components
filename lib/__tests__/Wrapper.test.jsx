import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-styled-components';
import Wrapper from '../elements/Wrapper';

describe('Wrapper', () => {
  it('renders correctly', () => {
    const component = renderer.create(<Wrapper />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});