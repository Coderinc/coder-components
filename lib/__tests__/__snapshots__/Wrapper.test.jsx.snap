// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Wrapper renders correctly 1`] = `
.c0 {
  box-sizing: border-box;
  width: 100%;
  height: 100%;
  overflow-x: hidden;
  overflow-y: hidden;
}

<div
  className="c0"
/>
`;
